app/src/main/res/layout-sw320dp/activity_upload_activity.xml

```diff
@@ -21,156 +21,212 @@
             app:layout_constraintTop_toTopOf="parent"
             app:navigationIcon="@drawable/menu"/>
 
-        <TextView
```

Comment:
I hope this new layout will be nice !

Reply:
Haha yes, you can try it, I will make another PR in the week to improve other things.